cmd/wallet-sdk-gomobile/openid4ci/interaction_test.go

```diff
@@ -84,6 +84,21 @@ func TestNewInteraction(t *testing.T) {
 		require.NotNil(t, interaction)
 	})
 
+	t.Run("Success http timeout", func(t *testing.T) {
+		kms, err := localkms.NewKMS(localkms.NewMemKMSStore())
+		require.NoError(t, err)
+
+		resolver := &mockResolver{keyWriter: kms}
+
+		requiredArgs := openid4ci.NewArgs(createTestRequestURI("example.com"), kms.GetCrypto(), resolver)
+		opts := openid4ci.NewOpts()
+		opts.SetHTTPTimeoutNanoseconds(10000000)
```

Comment:
For readability, I suggest passing in an equivalent time value from the time package instead of `10000000`

Reply:
dene